### PR TITLE
Add ability to communicate with elasticsearch using a service token

### PR DESCRIPTION
**Pull request by blakerouse**
## What does this PR do?

Adds the ability to communicate with elasticsearch using a service token.

Set the `service_token` key in the `output` defined in `fleet-server.yml`.

## Why is it important?

So Fleet Server can communicate using a service token versus using a username and password.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -29,5 +29,6 @@ require (
 replace (
 	github.com/Shopify/sarama => github.com/elastic/sarama v1.19.1-0.20200629123429-0e7b69039eec
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
+	github.com/elastic/go-elasticsearch/v8 => github.com/blakerouse/go-elasticsearch/v8 v8.0.0-20210413134915-cee76d43c5cf
```
Comment (ruflin) [inline go.mod:32]: @blakerouse Should we get it in directly with this "fork"? I think it is unlikely that we can use a stable release soon enough.

Comment (blakerouse) [reply go.mod:32]: @ruflin really? I just pushed this so others can test it. I didn't think getting it merged into go-elasticsearch was going to be a big deal. Its a very small change.

Comment (ruflin) [reply go.mod:32]: Looks like I was wrong and it is already in!

Comment (blakerouse) [reply go.mod]: Yep, updated to use new upstream version.

**Review by aleksmaus — APPROVED** [review]

**Pull request merged**